.github/workflows/docs/intro.txt

```diff
@@ -1,8 +1,8 @@
 pytket-extensions
 =================
 
-.. image:: CQCLogo.png
-   :width: 120px
+.. image:: Quantinuum_logo.png
+   :width: 200px
```

Comment:
Have you checked if the bigger image is displayed in the right way?

Reply:
Yes I have built the docs locally and it seems like an okay size. Do you think it looks okay?

Reply:
I have not taken a look yet, will build it now, too.

Reply:
Are we allowed within the CI to make the "Q" a bit bigger and keep the "quantinuum" in the same size?